canonical-web-and-design/maas.io
# Move 2.9 and 3.0 redirects to maas.io

Also checking whether transliteration is going on when the PR is posted/tested.  Took screenshots of "Preview changes" to verify no collapsed redirects (two left-hand colon stanzas in one line).  Will see what comes out the other side

## Done

Added 2.9 and 3.0 redirects to maas.io redirects.yaml, since removing redirects seems to dramatically speed up doc delivery.  Only keeping 3.1 and later redirects, will practice moving everything older than latest version on release.

## QA

Check that it works, of course.  Also, check for lines like this one:

docs/something/something: docs/someother/lefthandside: /docs/target/url

The last two PRs have had numerous errors like this, even though I thought I checked it.  I have screenshots of the changes before and after PR, showing that the file doesn't have these errors (that I can see).  Something in the process might be corrupting these?


## Issue / Card

[List of links to Github issues/bugs and cards if needed - e.g. `Fixes #1`]

## Screenshots

[if relevant, include a screenshot]
